dashboard/src/components/AppUpgrade/AppUpgrade.tsx

```diff
@@ -77,11 +87,33 @@ class AppUpgrade extends React.Component<IAppUpgradeProps, IAppUpgradeState> {
             resource="App Repositories"
           />
         );
+      } else if (repos.length === 0) {
+        return (
+          <ErrorSelector
+            error={new Error("Unable to any repo to upgrade from")}
```

Comment:
We should use an error similar to https://github.com/kubeapps/kubeapps/blob/714fff14ef4fe58358b51add4310ceb9a95f66fc/dashboard/src/components/Catalog/Catalog.tsx#L51-L60 in this case, so we can prompt users to adding repos.